spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/common/ConfigCommands.java

```diff
@@ -216,8 +216,7 @@ public String target(
 		try {
 			this.targetHolder.setTarget(new Target(targetUriString, targetUsername, targetPassword, skipSslValidation));
 
-			final HttpClientConfigurer httpClientConfigurer = HttpClientConfigurer.create()
```

Comment:
I think we have an issue with the `proxy-username` option configuration here. 
```
 			@CliOption(mandatory = false, key = {
 					"proxy-username" }, help = "the username for authenticated access to the secured proxy server (valid only with a "
 					+ "username)", specifiedDefaultValue = Target.DEFAULT_SPECIFIED_PASSWORD, unspecifiedDefaultValue = Target.DEFAULT_UNSPECIFIED_PASSWORD) String proxyUsername,
```

Reply:
Made the help wording better